src/main/java/net/dv8tion/jda/api/interactions/commands/localization/ResourceBundleLocalizationFunction.java

```diff
@@ -57,19 +57,59 @@ public Map<DiscordLocale, String> apply(@Nonnull String localizationKey)
         return map;
     }
 
+    /**
+     * Provides a {@link ResourceBundleLocalizationFunction} builder with the provided bundle having the specified name and locale.
```

Comment:
"having the specified name"
What name? 😕

Reply:
This description leaves much to the imagination in regards to what it actually is doing.

Reply:
Also, try starting with "Creates a X" instead of "Provides a X"

Reply:
Some examples of what actual usage would look like would be nice for these.
Kinda like what you did for `#addBundle`